synapse/handlers/federation.py

```diff
@@ -263,6 +264,7 @@ async def on_receive_pdu(
         state = None
 
         # Get missing pdus if necessary.
+        # We don't need to worry about outliers because TODO!
```

Comment:
Unrelated to the PR but just want to better clarify why we skip outliers in this step. Any insight here?

 - Outliers are outside of the normal DAG
 - We don't have state for outliers
 - We trust their auth events over calculating them